src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    static final ColorSupplier colorSupplier = new ColorSupplier();
+    static final int maxParameterValue = 50;
+    static final int FIGURES_COUNT = 5;
+    static final Random random = new Random();
+    static final int DEFAULT_RADIUS = 10;

```

Comment:
Let's try this:
```suggestion
    private static final int MAX_PARAMETER_VALUE = 50;
    private static final int FIGURES_COUNT = 5;
    private static final int DEFAULT_RADIUS = 10;

    private final ColorSupplier colorSupplier = new ColorSupplier();
    private final Random random = new Random();
```

The first group is your constants named accordingly to conventions in UPPER_SNAKE_CASE.

The second group is dependencies, which will not be changed (`final`), but they are not constants, because should be created for every FigureSupplier object.